tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,441 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "photlam_hatp11 = SourceSpectrum(Empirical1D,\n",
+    "                                points=wavelengths, lookup_table=flux)\n",
+    "\n",
+    "# Compute the flux at Earth from the luminosity \n",
+    "R = 0.683 * u.R_sun  # radius in solar units, Deming et al. 2011\n",
+    "d = 123 * u.lyr # distance to HATP11 in light years from Gaia parallax\n",
+    "photlam_hatp11 = photlam_hatp11 * float(R / d) ** 2 / np.pi\n",
+    "\n",
+    "photlam_hatp11.plot(flux_unit='Jy', left=0, right=60000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(for other methods of constructing source spectra with `synphot`, see the first bulleted list <a href=\"https://synphot.readthedocs.io/en/latest/synphot/spectrum.html#arrays\">here</a>) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Simulate the observation with a specific instrument"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Let's observe HAT-P-11 with the <a href=\"https://www.apo.nmsu.edu/arc35m/Instruments/ARCTIC/\">ARCTIC instrument</a> on APO's 3.5m telescope."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 131,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r = 3.5 / 2 * u.m  # radius of 3.5m ARC telescope at APO\n",
```

Comment:
In my next few comments, I hope you'll forgive me as I get a bit nitpicky about variable names. Zen of Python tells us
```
Explicit is better than implicit...
Readability counts.
```
In that spirit, and in keeping with the [astropy-tutorials examples](https://github.com/astropy/astropy-tutorials/blob/master/tutorials/notebooks/coordinates/coordinates.ipynb), let's give variables descriptive, full-word names. So:
```
r -> aperture_radius
R -> stellar_radius
```
etc.

Reply:
Good to know - I prefer being more explicit but was worried about being too "wordy"!

Reply:
I'm definitely pro-"wordy". The closer the code reads to an english sentence the better!